guess-the-country/country-backend/country/explainer/explain.py

```diff
@@ -40,6 +42,13 @@ def convert_explanation(explanation):
 
 
 @traced
-def explain_cnn(img, model):
-    return explain_image(img=img, seg_method="felzenszwalb", seg_settings={}, num_of_samples=500, samples_p=0.9,
-                         model_=model, threshold=0.5, volume=20, colour="violet", transparency=0.7)
+def explain_cnn(image, model):
+    def _predict_fn(img):
+        IMG_SIZE = 224
```

Comment:
Diese Konstante sollte nicht innerhalb der Funktion, sondern mindestens auf Modul-Ebene oder besser noch in predict.py einmal für den kompletten Use-Case definiert und dann hier importiert werden. Das ist ja eine Eigenschaft des Modells.